src/tests/cli/test_logout.py

```diff
@@ -8,8 +8,17 @@
 from ..base import IntegrationTests
 
 
+@unittest.skip("Only used in testenv:slow")
```

Comment:
Discussion: `skipIf` is more standard (and easier to maintain) here. May be worth the extra work to switch.

Reply:
Possible solution - skipIf on an ENV variable here, leave `tox` env out of the loop, set the ENV variable in CI/CD.

Need to document it for local dev - and ideally default to the fast version.